addons/logistics_wirecutter/script_component.hpp

```diff
@@ -50,7 +50,10 @@
     "wall_fen1_5.p3d",\
     "plot_provizorni.p3d",\
     "plp_ctm_partitioningfencegrey.p3d",\
-    "fence.p3d"\
+    "fence.p3d,"\
```

Comment:
I think it should be `"fence.p3d",` (coma at the end of line) no ?

Reply:
yes, after the " not before ;)